src/cli/playbooks/demo-script.sh

```
@@ -0,0 +1,127 @@
+
+# Parse parameters
+part=$1
+echo "on on $part"
+
+
+if [ "$part" = 1 ] || [ "$part" = 0 ]; then
+  # Initialize domain and setup
+  pnpm cli distributions add -i 1 -y
+  sleep 5
+  #move to contracts folder and run the script, then move back to sdk folder
+  cd ../contracts && pnpm hardhat --network localhost makeDemoSubjects && cd ../sdk 
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)